netsim/ansible/templates/bgp/frr.j2

```diff
@@ -56,12 +56,10 @@ router bgp {{ bgp.as }}
   network {{ pfx|ipaddr('0') }}
 {%   endfor %}
 !
-{%   for n in bgp.neighbors
-       if n[af] is defined 
-         and (n.activate[af] is defined and n.activate[af]) %}
-{%     set peer = n[af] if n[af] is string 
+{%   for n in bgp.neighbors if n.activate.get(af) %}
+{%     set peer = n[af] if n.get(af) is string
```

Comment:
It's still not a good idea to keep insisting on using an attribute for something it was NOT designed to do, but whatever.